plugins/domains/commands

```diff
@@ -65,7 +65,7 @@ case "$1" in
       exit 1
     fi
 
-    if [[ $(egrep ^"$3"$ "$DOKKU_ROOT/$APP/VHOST" > /dev/null 2>&1; echo $?) -eq 0 ]]; then
+    if [[ $(grep --fixed-strings "$3" "$DOKKU_ROOT/$APP/VHOST" > /dev/null 2>&1; echo $?) -eq 0 ]]; then
```

Comment:
This is a related bug that I caught, \*.example.com matches www.example.com. Since it's one domain per line, using fixed strings allows us to match the entire line, including \*'s. grep because it's better supported and threw less errors in testing.